src/resources/cluster.ts

```diff
@@ -1,107 +1,172 @@
+                        "GroupId": {
+                            "Ref": this.getName(NamePostFix.CONTAINER_SECURITY_GROUP)
+                        },
+                        "IpProtocol": -1,
+                        "SourceSecurityGroupId": {
+                            "Ref": this.getName(NamePostFix.CONTAINER_SECURITY_GROUP)
+                        }
+                    }
+                },
+                ...this.generateServicesSecurityGroups()
+            };
+        }
+    }
+
+    private getSecurityGroupNameByService(service: Service): string {
+        return `${this.getName(NamePostFix.LOAD_BALANCER_SECURITY_GROUP)}_${service.getName(NamePostFix.SERVICE)}`;
+    }
+    
+    private generateServicesSecurityGroups(): object {
+        let secGroups = {};
+        this.services.forEach( (service: Service) => {
+            secGroups = {
+                ...secGroups,
+                ...this.generateSecurityGroupsByService(service)
+            };
+        });
+        return secGroups;
+
+    }
+
+    private generateSecurityGroupsByService(service: Service): any {
+        const ELBServiceSecGroup = this.getSecurityGroupNameByService(service);
+        return {
+            //Public security groups
+            ...(this.options.public ? {
+                [ELBServiceSecGroup]: {
+                    "Type": "AWS::EC2::SecurityGroup",
+                    "DeletionPolicy": "Delete",
+                    "Properties": {
+                        ...(this.getTags() ? { "Tags": this.getTags() } : {}),
+                        "GroupDescription": `Access to the public facing load balancer - task ${service.getName(NamePostFix.SERVICE)}`,
+                        "VpcId": this.getVPC().getRefName(),
+                        "SecurityGroupIngress": [
+                            {
+                                "CidrIp": "0.0.0.0/0",
+                                "IpProtocol": -1,
+                                "toPort": service.port
+                            }
+                        ]
+                    }
+                },
+                [this.getName(NamePostFix.SECURITY_GROUP_INGRESS_ALB)]: {
+                    "Type": "AWS::EC2::SecurityGroupIngress",
+                    "DeletionPolicy": "Delete",
+                    "Properties": {
+                        "Description": `Ingress from the ALB - task ${service.getName(NamePostFix.SERVICE)}`,
+                        "GroupId": {
+                            "Ref": this.getName(NamePostFix.CONTAINER_SECURITY_GROUP)
+                        },
+                        "IpProtocol": -1,
+                        "SourceSecurityGroupId": {
+                            "Ref": ELBServiceSecGroup
+                        }
+                    }
+                }
+            } : {
+                /*TODO: if not public AND also not specifiying a VPC, different secgroup must be created*/
```

Comment:
We should console.err this or something, so its very obvious that this is not being created.